servant/CHANGELOG.md

```diff
@@ -1,5 +1,73 @@
 [The latest version of this document is on GitHub.](https://github.com/haskell-servant/servant/blob/master/servant/CHANGELOG.md)
 
+0.13
+----
+
+### Significant changes
+
+- Streaming endpoint support.
+  ([#836](https://github.com/haskell-servant/servant/pull/836))
```

Comment:
Might be nice to link to the relevant section in the tutorial?

Reply:
How to make `readthedocs` build branches?

Reply:
found.